### PR TITLE
[mod] check todo list.

### PR DESCRIPTION
**Pull request by M5oul**
Tell me I am wrong at some place.

### CODE REVIEW

**Comment by lsimonne** [comment]
starting to look good!
we could add history_substitution somewhere, still needs a bit of testing

File: README.md [hunk 1]
```diff
@@ -13,30 +13,30 @@ FEATURES
...
-	- [ ] Les redirection suivantes : ">", ">>", "<".
-	- [ ] Les pipes “|”.
-	- [ ] Les opérateurs logiques "&&" et "||".
-	- [ ] Les back quotes "‘".
-	- [ ] Le séparateur ";".
+	- [x] Exécution de commandes avec leurs paramètres et gestion du PATH.
+	- [x] Gestion des erreurs et de la valeur de retour des commandes.
+	- [x] Les redirection suivantes : ">", ">>", "<".
+	- [x] Les pipes “|”.
+	- [x] Les opérateurs logiques "&&" et "||".
```
Comment (lsimonne) [inline README.md]: not sure on where we are on this

Comment (M5oul) [inline README.md]: I tested and it works fine.
May be I am wrong.

Comment (lsimonne) [inline README.md]:
```
sh-3.2$ ls && false || echo PASS
42sh		README.md	libft		srcs		tools
42sh.dSYM	auteur		objs		testcomp
Makefile	includes	set_histfile.sh	tests
PASS
```

```
[ChezLouise@ ~/Documents/42/42sh]$ ls && false || echo PASS
42sh		README.md	libft		srcs		tools
42sh.dSYM	auteur		objs		testcomp
Makefile	includes	set_histfile.sh	tests
```

par contre ça marche avec un builtin:

```
[ChezLouise@ ~/Documents/42/42sh]$ echo PASS && false || echo PASS
    executing builtin echo
PASS
    done executing builtin echo, ok
    executing builtin echo
PASS
    done executing builtin echo, ok
```

**Pull request merged**